<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskSetWidget</class>
 <widget class="QWidget" name="TaskSetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser_taskSeq">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>211</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_planRes">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>任务序列</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>添加任务</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>删除任务</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>任务规划</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addTask">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_deleTask">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_startPlan">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>204</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始规划</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_addTask">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_deleTask">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
